tool-plugins/intellij/src/main/java/icons/layout/code.svg

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+ ~   Copyright 2017, WSO2 Inc. (http://www.wso2.com)
```

Comment:
Incorrect year? Can be updated in a few other files as well.

Reply:
Fixed the year.